### PR TITLE
feat(pages): import all subdomains to pages

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

File: client/src/pages/about.js
Comment (ValeraS) [inline client/src/pages/about.js]: @raisedadead No need to set `external={true}` for absolute links

Comment (raisedadead) [reply client/src/pages/about.js]: See [this diff for what we are upto](https://github.com/freeCodeCamp/freeCodeCamp/pull/35243/files/7e827a27744d1edced2e6e2cb051dc429dbb076a#diff-8cc13ecb83eae1c4cc561432bcfd97d9), I borrowed this from our footer logic and moved to a helper util.

Comment (raisedadead) [reply client/src/pages/about.js]: Most of the time we would want to open links in new tabs, other than our own pages. Sometimes our own pages may suite better to be opened in new tabs.

This is just to unify some of it.

This commit explains the use-case: https://github.com/freeCodeCamp/freeCodeCamp/pull/35243/commits/7e827a27744d1edced2e6e2cb051dc429dbb076a

Comment (ValeraS) [reply client/src/pages/about.js]: Yes, I saw it.

Comment (ValeraS) [reply client/src/pages/about.js]: > Most of the time we would want to open links in new tabs, other than our own pages. Sometimes our own pages may suite better to be opened in new tabs.

Yes, and `external` in this component is used for making local links behave as external.

Comment (raisedadead) [reply client/src/pages/about.js]: Oh Fish! Right, I misunderstood what you meant. Thanks, will fix it.

File: client/src/pages/support.js
Comment (ValeraS) [inline client/src/pages/support.js]: We can't use `/forum` links without `external={true}` because of Gatsby don't know about them.

File: client/src/pages/code-of-conduct.js
Comment (ValeraS) [inline client/src/pages/code-of-conduct.js]: No need to set `className='container'`, it is used by default for `Grid`

Comment (raisedadead) [reply client/src/pages/code-of-conduct.js]: This PR is in draft. :)

All I am doing right now, is porting over the html to react. There is a fair chance that I'll miss a ton of things. I'll go through the code once again once I am ready

**Comment by raisedadead** [comment]
I have one last thing to work on this PR, which is linting this with prettier for the formating. I am waiting for @Bouncey's PR to land.

@ValeraS Thanks a lot for your early feedback. I think this is ready for a QA.

File: client/src/pages/support.js
Comment (ValeraS) [inline client/src/pages/support.js]:
```suggestion
              <Link external={true} to='/forum/new-topic?category=support'>
```

File: client/src/pages/about.js
Comment (ValeraS) [inline client/src/pages/about.js]:
```suggestion
              , and post on <Link external={true} to='/forum'>our forum</Link>, each month we
```

**Review by ValeraS — CHANGES_REQUESTED** [review]

File: client/src/pages/index.js
Comment (Bouncey) [inline client/src/pages/index.js]: Would this work as a `<Link>` now it is a relative href?

File: client/src/pages/learn.js
Comment (Bouncey) [inline client/src/pages/learn.js]: Would this work as a `<Link>` now it is a relative href?

File: client/src/pages/sponsors.js
Comment (Bouncey) [inline client/src/pages/sponsors.js]: We can still enforce the `max-len` if you rearrange the props to use string concatenation, like so:

```jsx
<Helmet
  title={
    'Sponsors who help freeCodeCamp through financial and in-kind ' +
    'sponsorship | freeCodeCamp.org'
  }
/>
```

**Review by Bouncey — CHANGES_REQUESTED** [review]
Use string concatenation to re-enable the `max-len` rule. I have provided an example

**Comment by raisedadead** [comment]
I think we are good to go once CI is done.

**Review by Bouncey — APPROVED** [review]

**Pull request merged**
